### PR TITLE
fix: add backwards compat for new env var and small typo

### PR DESCRIPTION
**Pull request by d1onys1us**

### CODE REVIEW

File: docker-compose.l3.yml [hunk 1]
```diff
@@ -123,7 +123,7 @@ services:
       - -c
       - "/script/start-zkevm-chain-rpcd.sh"
     deploy:
-      replicas: ${ZKEVM_CHAIN_INSTANCES_NUM}
+      replicas: ${ZKEVM_CHAIN_INSTANCES_NUM:-1}
```
Comment (davidtaikocha) [inline docker-compose.l3.yml:126]: emm shall we use `1` here as the default `replicas` value?

Comment (d1onys1us) [reply docker-compose.l3.yml:126]: yes it is 1 right? or i might have the syntax wrong if chatGPT lied.

Comment (d1onys1us) [reply docker-compose.l3.yml:126]: like if it is not set, then it will be 1, i believe.

Comment (davidtaikocha) [reply docker-compose.l3.yml:126]: Ah yeah `:-` is the way to set default value in docker compose, i thought its `:`...

**Review by davidtaikocha — APPROVED** [review]

**Pull request merged**